### PR TITLE
TFTP protocol

### PR DESCRIPTION
**Pull request by earlgreyz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,8 +102,8 @@ including a customizable histogram of latencies, an error rate and some other su
 Bender does provide a complete log of everything that happens during a load test, so you can use
 existing tools to graph any aspect of that data, but nothing in Bender makes that easy right now.
 
-Bender currently provides helper functions for HTTP, Thrift and DNS. We appreciate PR for other
-protocols.
+Bender currently provides helper functions for HTTP, Thrift, DNS and TFTP. We appreciate PR for
```
Comment (jparise) [inline README.md]: Let's spell out "Pull Requests" here. I think that's a little clearer.

File: tftp/tftp.go [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+type Request struct {
+	Filename string
+	Mode     RequestMode
+}
+
+// ResponseValidator validates a udp response.
+type ResponseValidator func(request *Request, writer io.WriterTo) (interface{}, error)
+
+// DiscardValidator reads the whole request and discards its body.
+func DiscardValidator(request *Request, writer io.WriterTo) (interface{}, error) {
```
Comment (jparise) [inline tftp/tftp.go]: What do you think about naming this `DiscardingValidator`?

**Comment by earlgreyz** [comment]
@cgordon fixed

**Comment by cgordon** [comment]
Looks good, thanks!

**Pull request merged**